### ISSUE TITLE
Allow shorter syntax for tasks with default configuration

### ISSUE THREAD

**Issue by andreynering** [go-task/task#194]
Instead of requiring tasks to be declared as:

```yaml
version: '2'

tasks:
  foobar:
    cmds:
      - echo "something"
```

We could allow:

```yaml
version: '3'

tasks:
  foobar:
    - echo "some"
    - echo "thing"
```

Or even:

```yaml
version: '3'

tasks:
  foobar: echo "something"
```

If users want to change any task configuration, they'd have to use the original form.

**Comment by Ilyes512** [comment]
I don’t actually mind having more verbose task. What I do miss is the option to configure `silent: true` globally instead of having to repeat it everywhere.

**Comment by andreynering** [comment]
This was just implemented on the `v3` branch in b7b752b92f5cb2b38a929eb447b36cf5fc35c80e.

**Comment by andreynering** [comment]
> What I do miss is the option to configure `silent: true` globally instead of having to repeat it everywhere.

@Ilyes512 Sorry, I forgot about your suggestion for a long time, but this is now done at 4bdfe64afb6cf96c4daa1faacf06b036864506e7.

For future feature requests, it's better to open new issues so it's less likely it'll be forgotten.
